modules/cluster-logging-log-forwarding-configure.adoc

```diff
@@ -85,7 +85,7 @@ spec:
 ----
 <1> The name of the log forwarding CR must be `instance`.
 <2> The namespace for the log forwarding CR must be `openshift-logging`.
-<3> Set to `enabled` to enable log forwarding.
+<3> Set to `true` to enable log forwarding.  
```

Comment:
Same comment as previously

Reply:
Changed to:
`<3> Set to `true` to enable log forwarding to the specified outputs. `

At this point, I assume that users are not as concerned with turning off the default behavior, just how to make the log forward feature work.

Reply:
@jcantrill In the Disabling section, I have: 
`Setting disableDefaultForwarding=false prevents cluster logging from sending logs to the specified endpoints and to default internal OpenShift Container Platform Elasticsearch instance.` 
Based on https://bugzilla.redhat.com/show_bug.cgi?id=1818034, which doesn't sound like will get fixed. If we want this info here, do we want it earlier when I introduce the parameter?